src/sgd/tools/blast/__init__.py

```diff
@@ -26,7 +27,7 @@ def _run_blast(p):
 
     paramData = _construct_blast_parameters(p)
 
-    url = config.compute_url + "cgi-bin/blast2.pl"
+    url = config.compute_url + "cgi-bin/aws-blast"
```

Comment:
Is the domain/hostname of the new server setup to match this? will ask @stuartmiyasato.

Reply:
Does it make sense to go ahead and setup the production BLASt server before we ask people to test? I know we usually do QA things in dev account, but maybe this is an exception?